module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -304,7 +304,7 @@ protected function setCommonSettings($searchSettings, $setting, $list, $target)
         if (isset($searchSettings->General->$setting)) {
             $userValues = explode(',', $searchSettings->General->$setting);
 
-            if (!empty($userValues) && isset($this->$list) && !empty($this->$list)) {
+            if (isset($this->$list) && !empty($this->$list)) {
```

Comment:
Can we also eliminate the isset here since that is implied by the empty check?

Reply:
Sure